src/bci_build/package/appcontainers.py

```diff
@@ -249,6 +249,93 @@
         )
     )
 
+
+_POSTFIX_FILES = {}
+for filename in (
+    "entrypoint.sh",
+    "entrypoint.sles.sh",
+    "postfix-manpages.md",
+    "smtpd_sender_login_maps",
+    "virtual_alias_domains",
+    "virtual_alias_maps",
+    "virtual_gid_maps",
+    "virtual_mailbox_maps",
+    "virtual_uid_maps",
+):
+    if filename.startswith("entrypoint.") or filename.endswith(".md"):
+        if os_version != OsVersion.TUMBLEWEED and filename == "entrypoint.sles.sh":
+            _POSTFIX_FILES["entrypoint.sh"] = (
+                Path(__file__).parent / "postfix" / "entrypoint" / filename
+            ).read_bytes()
+        else:
+            _POSTFIX_FILES[filename] = (
+                Path(__file__).parent / "postfix" / "entrypoint" / filename
+            ).read_bytes()
+    else:
+        _POSTFIX_FILES[filename] = (
+            Path(__file__).parent / "postfix" / "entrypoint" / "ldap" / filename
+        ).read_bytes()
+
+
+POSTFIX_CONTAINERS = [
+    ApplicationStackContainer(
+        name="postfix",
+        package_name="postfix-image",
+        pretty_name="Postfix",
+        custom_description="Postfix container is fast and secure mail server, {based_on_container}.",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%postfix_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%postfix_version%%",
+                package_name="postfix",
+                parse_version="minor",
+            )
+        ],
+        package_list=[
+            name
+            for name in (
+                "ca-certificates-mozilla",
+                "cyrus-sasl",
+                "cyrus-sasl-plain",
+                "ed",
+                "netcfg",
+                "postfix",
+                "postfix-ldap",
+                "postfix-lmdb",
+                "timezone",
+            )
+            + (
+                (
+                    "mandoc",
+                    "spamassassin-spamc",
+                    "spamass-milter",
+                )
+                if os_version == OsVersion.TUMBLEWEED
+                else ()
+            )
+        ],
+        entrypoint=["/entrypoint/entrypoint.sh && cat /entrypoint/postfix-manpages.md"],
```

Comment:
This does not work, the entrypoint is not passed via a shell but executed directly
```suggestion
        entrypoint=["/entrypoint/entrypoint.sh"],
```

Reply:
moving the text in `postfix-manpages.md` to `entrypoint.sles.sh` as a echo command, and removing the md file after.